rules/windows/registry_event/sysmon_modify_screensaver_binary_path.yml

```diff
@@ -0,0 +1,28 @@
+title: Path To Screensaver Binary Modified
+id: 67a6c006-3fbe-46a7-9074-2ba3b82c3000
+status: experimental
+description: Detects value modification of registry key containing path to binary used as screensaver.
+references:
+  - https://attack.mitre.org/techniques/T1546/002/
```

Comment:
Hello @bczyz1 ! The same thing here (: Please remove the link to ATT&CK here. Everything else is just fine! Thank you

Reply:
Hey @yugoslavskiy, done in ae41190291a81e2b8d1f1e1dff28e8354bb63178. Thanks for review!